Lib9c/Arena/ArenaSimulator.cs

```diff
@@ -73,8 +74,18 @@ public ArenaSimulator(IRandom random, int hpModifier = 2)
 
                 foreach (var other in players)
                 {
+                    var spdMultiplier = 0.6m;
                     var current = players.GetPriority(other);
-                    var speed = current * (other.usedSkill != null ? 0.9m : 0.6m);
+                    var skills = sheets.SkillSheet.OrderedList.Select(s => s.SkillCategory);
+                    var skillCategory =
+                        other.usedSkill?.SkillInfos.Where(si => skills.Contains(si.SkillCategory))
+                            .Select(si => si.SkillCategory);
+                    if (skillCategory != null && skillCategory.Any(sc => sc != SkillCategory.NormalAttack))
```

Comment:
it can be handled by checking the usedSkill is ArenaNormalAttack without check sheet.

---
시트를 조회할필요없이 usedSkill이 ArenaNormalAttack인지 확인하는걸로 처리가 가능할것같습니다.